src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -603,11 +621,13 @@ else if (workBuilding != building)
             }
             else if (job != null)
             {
-                final EntityCitizen citizen = getCitizenEntity();
-                if (citizen != null)
-                {
-                    citizen.tasks.removeTask(citizen.tasks.taskEntries.stream().filter(task -> task.action instanceof AbstractAISkeleton).findFirst().orElse(null).action);
-                }
+                getCitizenEntity().ifPresent(entityCitizen -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1602)